### PR TITLE
Feature/22 "hanging" issues

### PR DESCRIPTION
**Pull request by zsinnema**
see #22

### CODE REVIEW

**Comment by gidili** [comment]
Ready for @phildong's review

File: pipeline.ipynb [hunk 1]
```diff
@@ -72,7 +72,9 @@
     "dpath = \"./demo_movies\"\n",
     "subset = dict(frame=slice(0,None))\n",
     "subset_mc = None\n",
+    "\n",
```
Comment (phildong) [inline pipeline.ipynb:75]:
```suggestion
```

File: pipeline.ipynb [hunk 2]
```diff
@@ -72,7 +72,9 @@
...
     "interactive = True\n",
+    "\n",
```
Comment (phildong) [inline pipeline.ipynb:77]:
```suggestion
```

File: pipeline.ipynb [hunk 3]
```diff
@@ -216,12 +218,14 @@
    "outputs": [],
    "source": [
     "dpath = os.path.abspath(dpath)\n",
+    "hv.notebook_extension('bokeh')\n",
```
Comment (phildong) [inline pipeline.ipynb:221]: can we put this into a separate PR? we can deprecate `interactive` completely if we're sure bokeh is working fine in all cases.

File: pipeline.ipynb [hunk 4]
```diff
@@ -1459,6 +1467,7 @@
    "outputs": [],
    "source": [
     "if interactive:\n",
+    "    # \n",
```
Comment (phildong) [inline pipeline.ipynb:1470]:
```suggestion
```

File: pipeline.ipynb (hunk 3 above)
Comment (zsinnema) [reply pipeline.ipynb:219]: I would like to have "interactive" to run the notebook without the intermediate results to speed up the result (by skipping display only computations)

Comment (phildong) [reply pipeline.ipynb:219]: That sounds good, but I think this should belong to a separate PR since it's independent of the hanging issue? Other than this I think it's ready to merge!

**Comment by zsinnema** [comment]
I've reverted the matplotlib change
